flow/display_list.cc

```diff
@@ -1365,10 +1365,10 @@ void DisplayListBuilder::drawTextBlob(const sk_sp<SkTextBlob> blob,
 void DisplayListBuilder::drawShadow(const SkPath& path,
                                     const SkColor color,
                                     const SkScalar elevation,
-                                    bool occludes,
+                                    bool transparentOccluder,
```

Comment:
yeah, since I'm introducing this here I should get the name right. Still, I don't like "transparent_occluder". Why? Because the name is a not-name, or an oxymoron, or a self-contradicting name and it is why I ran into this in the first place. "this is a flag about occlusion!" "OK, so it hides something, right? That's what occlusion refers to." "No, um, this flag is about failing to occlude something." "But it has 'occlude' right there in its name???"

It would be less confusing if it had a "not" word in it, but you have to understand what transparency is and what occlusion is to notice that they contradict each other and so this is a flag about something that isn't there.

The record name was "NonOccludingOp". I'm trying to think of a better word.

Option B, reverse its meaning in DL, but then it would be negating its value going into and out of the DL stream and any code that was dispatching or being dispatched to would have to recognize that this usage is negated from the Skia usage.

Reply:
As mentioned in https://github.com/flutter/engine/pull/28753 I left it as transparent_occluder which at least follow our style guide on capitalization of parameters, but it leaves the questionable name in play. Since that naming is used all the way from Flutter classes through the PhysicalShapeLayer through to Skia, I decided to leave it in the DisplayList code, but with corrected format `transparent_occluder`.